src/raftstore/store/snap.rs

```diff
@@ -704,14 +707,22 @@ impl Snap {
             } else {
                 let mut key_count = 0;
                 let mut size = 0;
+                let base: i64 = limiter.get_min_bytes_per_time();
+                let mut bytes: i64 = 0;
                 snap.scan_cf(
                     cf,
                     &begin_key,
                     &end_key,
                     false,
                     &mut |key, value| {
+                        let l = key.len() + value.len();
```

Comment:
This can be read from cache, so it seems not very reasonable to use an io limiter here.
Maybe we can limit the write rate by adding a rate limiter to the underlying `SstFileWriter` instead?

Reply:
This is for sst writer, not for read.